README.md

```diff
@@ -1,7 +1,44 @@
 # Gift of Open Source
 
-- Welcome to Goos! Here’s how you get started
-- [Contributor's guide](https://github.com/pantheon-systems/gift-of-open-source/tree/master/content/contributors.md) - Start here if this is your first time here or need a refresher on how to get started.
-- [Maintainer's guide](https://github.com/pantheon-systems/gift-of-open-source/tree/master/content/maintainers.md) - Review these instructions if you are a code maintaniner and would like to participate
-- Live streams Schedule
-- If someone has a question, they can join community slack and specific channel.  Or they can open an issue (use templates https://github.com/pantheon-systems/documentation/issues/new/choose)
+Welcome! Help make the open web more inclusive, efficient and impactful, by contributing to the open source projects you love most, during the month of December. The [Gift of Open Source](https://pantheon.io/gift-open-source#supportopensource) is a month-long event that is a celebration of community, learning, giving back, and is free of charge.
+
+Read on to learn how you can get started.
+
+* [Participate](#participate)
+* [Engagement](#engagement)
+* [Questions](#questions)
+* [Additional Resources](#additional-resources)
+* [Acknowledgements](#acknowledgements)
+
+## Participate
+
+So, you want to get involved:
+
+- [Contributor's guide](https://github.com/pantheon-systems/gift-of-open-source/tree/master/content/contributors.md) - Start here if this is your first time here or need a refresher on how to get started. This document is for anyone who would like to learn how to contribute to the event and win great swag.
+
+- [Maintainer's guide](https://github.com/pantheon-systems/gift-of-open-source/tree/master/content/maintainers.md) - Review these instructions if you are a code maintaniner and would like to participate.
+
+
+## Engagement
+
+- Live streams Schedule 
+
+
+## Questions
+
+If you have any questions or concerns, you can send a message in the [Gift of Open Source](https://pantheon-community.slack.com/archives/C01D3VBLAHM) community Slack. You can also [submit an issue](https://github.com/pantheon-systems/gift-of-open-source/issues/new/choose) to this [repo](https://github.com/pantheon-systems/gift-of-open-source). 
+
+
+## Additional Resources
+
+* [Gift of Open Source](https://pantheon.io/gift-open-source)
+* [Registration](https://pantheon.io/gift-open-source#register)
+
+
+_____________________________
+
+## Acknowledgements
+
+We would like to give credit to Hacktoberfest 2021 for inspiring the text in this repository. 
```

Comment:
How about this instead:
Shout out to Hacktoberfest for inspiring this project and creating a model for the Gift of Open Source.